hearth/lib/hearth/validator.rb

```diff
@@ -41,5 +41,20 @@ def self.validate_unknown!(struct, params, context:)
       raise ArgumentError,
             "Unexpected members: [#{unknown.join(', ')}]"
     end
+
+    # draft of what validate_range *could* look like
+    # valid_range param consists of min and max keys
+    def self.validate_range(value, valid_range, context:)
+      # need to validate the valid_range object before proceeding
+      # does it make sense us to use existing method - validate_type!
+      valid_range.each { |context, value| self.validate_type!(value, Integer, context: context) }
```

Comment:
I think we can probably skip validating that the inputs for min/max are correct since these aren't intended as customer input.